client/blocks/edit-gravatar/style.scss

```diff
@@ -1,19 +1,27 @@
 .edit-gravatar__image-container {
 	display: inline-block;
 	position: relative;
-	line-height: 0;
 
 	&.is-uploading::before {
 		content: '';
 		position: absolute;
 			top: 0;
-			right: 0;
-			bottom: 0;
+			right: 0; // needed for IE11
+			bottom: 0; // needed for IE11
```

Comment:
Good question ;). Turns out this is incorrect. I removed `right: 0; bottom: 0;` and added styles for the fallback Gravatar which do help with IE11.